doc/man1/flux-cancel.rst

```diff
@@ -0,0 +1,56 @@
+.. flux-help-description: cancel one or more jobs
+.. flux-help-section: jobs
+
+==============
+flux-cancel(1)
+==============
+
+
+SYNOPSIS
+========
+
+**flux** **cancel** [*OPTIONS*] [*JOBIDS..*]
```

Comment:
Elipses has a missing period.

Maybe it would be good to use the form `[JOBID ...]` like in [flux-jobs(1)](https://flux-framework.readthedocs.io/projects/flux-core/en/latest/man1/flux-jobs.html)?

Reply:
Fixed and force-pushed the result.